.github/workflows/build-release.yml

```diff
@@ -8,32 +8,32 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@154c24e1f33dbb5865a021c99f1318cfebf27b32
+        uses: docker/setup-buildx-action@v3.0.0
```

Comment:
I'd rather we still point to commit hashes, for reproducibility sakes'. Tags can change, we can add a comment with the version for future happyness though.